src/asphalt/core/cli.py

```diff
@@ -67,6 +81,27 @@ def run(configfile, unsafe: bool, loop: Optional[str], service: Optional[str]) -
         ), "the document root element must be a dictionary"
         config = merge_config(config, config_data)
 
+    # Read the configuration from the CLI
+    for cli_conf in set_:
+        if "=" not in cli_conf:
+            click.echo(f"Configuration must be set with '=', got: {cli_conf}")
+            raise click.Abort()
+
+        key, value = cli_conf.split("=", 1)
+        keys = [k.replace("\\.", ".") for k in re.split("(?<!\\\\)\\.", key)]
```

Comment:
It's a bit more readable with raw strings.
```suggestion
        keys = [k.replace(r"\.", ".") for k in re.split(r"(?<!\\)\.", key)]
```

Reply:
:+1: